### PR TITLE
[ADD] gap_analysis_crm

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: gap_analysis_crm/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+{gap_analysis_crm}
+==============
+
+linking business requirements to opportunity.
+
+Installation
+============
+
+To install this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_crm/README.rst]: No specific step required for installation

File: gap_analysis_crm/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_crm/README.rst]: No specific step required for configuration

File: gap_analysis_crm/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```
Comment (elicoidal) [inline gap_analysis_crm/README.rst]: modify accordingly repo/module

File: gap_analysis_crm/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
```
Comment (elicoidal) [inline gap_analysis_crm/README.rst]: modify accordingly

File: gap_analysis_crm/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_crm}
+==============
+
+linking business requirements to opportunity.
```
Comment (elicoidal) [inline gap_analysis_crm/README.rst]: This module is part of a set ("Gap analysis") and extend the basic functionality
to link the business analysis to CRM opportunity.

**Comment by rafaelbn** [comment]
Please provide a description in PR and refer to the PR which this depends: https://github.com/OCA/project/pull/127. WIP till https://github.com/OCA/project/pull/127 is merged.

**Pull request closed without merging**